### PR TITLE
Fix UG Bugs & Update DG Diagrams

### PR DESCRIPTION
**Pull request by thienmy0**
# Summary of change
Fix UG bugs and update DG diagrams
---
# Key changes

## UG
1. Table of contents & Command Summary
2. Fix bugs

## DG
1. Update record sequence diagram
2. Add record activity diagram

### CODE REVIEW

File: docs/diagrams/RecordCommandParserActivityDiagram.puml
Comment (AaronJT1) [inline docs/diagrams/RecordCommandParserActivityDiagram.puml:1]: good to see that you used activity diagram for ur dg!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -361,7 +362,7 @@ Examples:
 * To add a patient with name "Betsy Crowe", IC number "S0123456B", phone number "90909090", and email "bc@gmail.com"<br>
     `add n/Betsy Crowe i/S0123456B p/90909090 e/bc@gmail.com`
 
-[Back to Table-of-contents](#table-of-contents)
+[Back to Table of Contents](#table-of-contents)
```
Comment (AaronJT1) [inline docs/UserGuide.md:365]: Thanks for standardising the `Back to Table Of Contents` link!

**Review by AaronJT1 — APPROVED** [review]
Nice to see that you have updated the UG and DG, LGTM!

**Review by wujy28 — APPROVED** [review]
LGTM!

**Pull request merged**